### DISCUSSION TITLE
Bug Bounty

### DISCUSSION THREAD

**Discussion by KGSSV** [community/community#109632] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

We have found version control bug.  Which severely impacts the Integrity of repository  ...
Can anyone guide us where to submit this issue for bug bounty ..

It seems hackerOne platform is only for security issues.. rather this is a application functionality issue

**Comment by namith1003** [comment]
The UX of github is really bad since microsoft took over they lost most links and even repositories where you couldve reported these kind of issues to, this doesnt count as a bug bounty since bug bounties mainly indicate security and vulnerability issues and if you feel llike the bug you found can be exploited and the integrity of the repository can be exposed, then use the link below for bug bounties for github.

https://bounty.github.com/

If not and it is indeed a general application functionality issue as you have stated then it doesnt really come under a bug bounty and im not sure if it is compensated for (im not sure it may be) but as of 2024 the only place to really submit bug issues like this is,

https://support.github.com/contact/bug-report


Hopefully, this helps you and you found the answer to your question, if you have anything else please don't mind contacting,

A.A Namith Nimlaka
